compliance/portugal.mdx

```diff
@@ -0,0 +1,157 @@
+
+<AccordionGroup>
+  <Accordion title="ACTUD/QR (B2B, B2G)">
+
+Although no format is required, all electronic invoices must include ATCUD and a QR code.
+
+1. **ATCUD (Unique Code)**: 
+   - Businesses must register their invoice numbering series with the AT
+   - The AT issues a validation code for each registered series
+   - This code must appear on each invoice
+
+2. **QR Code**:
+   - All e-invoices must contain a QR code
+   - The QR code links to the AT's online verification service
+
+
+  |            |             |
+  |------------|-------------|
+  | **Models** | B2B |
+  | **Format** | EN 16931 with CUIS extensions |
+  | **Infrastructure** | Hybrid: public and private platforms |
+  | **Model** | Centralized |
+  | **Scope & Deadline** | All B2B invoices from **Jan 1, 2026**; mandatory for suppliers to public authorities. Optional for B2C simplified invoices. |
+  | **Agency** | [AT Portugal](https://www.portaldasfinancas.gov.pt) |
+  | **Invopop Support** | [AT Portugal App](/apps/at-pt)<br />[InvoiceXpress Portugal App](/apps/pt-invoicexpress) |
+
+
+    <Columns cols="2">
+      <Card title="AT Portugal" size="20" icon="https://assets.invopop.com/apps/at-pt/icon.svg" href="/apps/at-pt" horizontal>Get covered →</Card>
+      <Card title="InvoiceXpress Portugal" icon="https://assets.invopop.com/apps/invoicexpress/icon.svg" href="/apps/pt-invoicexpress" horizontal>Get covered →</Card>
+    </Columns>
+
+  </Accordion>
+</AccordionGroup>
+
+## E-reporting
+
+The SAF-T (Standard Audit File for Tax) system was introduced in Portugal in 2008 to facilitate the electronic submission of VAT returns, accounting records, and transport documents. This system requires files to be generated in specific formats containing detailed information regarding commercial transactions. Taxpayers with headquarters, based or permanently established in Portugal, subject to corporate tax (IRC) or income tax (IRS) that keep organized accounting, must comply with this system.
+
+
+<AccordionGroup>
+  <Accordion title="SAF-T (PT) (B2B, B2G)">
+
+    SAF-T (Standard Audit File for Tax) is an XML-based file format used to report accounting and invoicing data to the tax authority, enabling standardized digital audits and tax compliance.
+
+
+    |                     |                                               |
+    |---------------------|-----------------------------------------------|
+    | **Format**          | SAF-T PT (XML)                                     |
+    | **Scope&#160;&&#160;deadline**|  B2G and B2B from 1 Jan 2024 | 
+    | **Invopop support**| Yes |
+
+
+    <Card title="AT Portugal" size="20" icon="https://assets.invopop.com/apps/at-pt/icon.svg" href="/apps/at-pt" horizontal>Get covered →</Card>
+
+  </Accordion>
+</AccordionGroup>
+
+## Regulation
```

Comment:
Just a general thought, this section feels a bit arbitrary regarding the contents that covers and what it lefts out. A bit like a FAQ. I guess we can add more info and structure in the future.